### PR TITLE
Update README.md with improved formatting and grammar and instructions

### PR DESCRIPTION
**Pull request by ismailbentabett**
## PR Checklist
Please ensure that your PR fulfills the following requirements:

- [x] The commit message follows our guidelines, as detailed in [CONTRIBUTING.md](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit).
- [ ] Tests have been added for the changes (for bug fixes/features).
- [] Docs have been added/updated (for bug fixes/features).

## PR Type
Please specify the type of change this PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] CI-related changes
- [ ] Documentation content changes
- [ ] angular.io application/infrastructure changes
- [ ] Other... (Please describe)

## Current Behavior
Please describe the current behavior that you are modifying or link to a relevant issue.

Issue Number: N/A

## New Behavior
Please describe the new behavior introduced by this PR.

## Breaking Change
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and provide a migration path for existing applications below.

## Other Information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<h1 align="center">Angular - The modern web developer's platform</h1>
+<h1 align="center">Angular - The Modern Web Developer's Platform</h1>
```
Comment (JeanMeche) [inline README.md:1]: I don't think this is meant to be capitalized.

File: README.md [hunk 2]
```diff
@@ -109,7 +110,7 @@ Angular is cross-platform, fast, scalable, has incredible tooling, and is loved
 
 ## Upgrading
 
-Check out our [upgrade guide](https://update.angular.io/) to find out the best way to upgrade your project.
+Check out our [upgrade guide](https://update.angular.io/) for the best way to upgrade your project.
```
Comment (JeanMeche) [inline README.md:113]: I feel like the previous formulation was better.

**Comment by bencodezen** [comment]
@ismailbentabett I wanted to check in to see if you're able to make updates as mentioned in the review. If not, I'll be closing the PR in the next week. Thanks!

**Comment by josephperrott** [comment]
Closing this PR as stale, feel free to open a different PR if you want to propose changes

**Pull request closed without merging**